cromwell-backend/src/main/scala/cromwell/backend/io/shared/SharedFileSystemIoInterface.scala

```diff
@@ -0,0 +1,56 @@
+package cromwell.backend.io.shared
+
+import java.nio.file.{Files, Paths}
+
+import cromwell.backend.io.IoInterface
+
+import scala.language.postfixOps
+import scala.util.Try
+
+object PathUtil {
+
```

Comment:
Removed superfluous \n and yes... you will see functionality copied from Cromwell since we want to get rid of utils and for that if we identify some func that belongs to IO in Cromwell-Backend, we move to this library.